project/fr-FR/Mod/No Pollution.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Options.SECTION[NoPollution.NoPollution.Mod]": "No Pollution",
+  "Options.GROUP[NoPollution.NoPollution.Mod.Resetting]": "Réinitialiser la pollution sonore",
+  "Options.OPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollutionReset]": "Pollution sonore",
+  "Options.OPTION_DESCRIPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollutionReset]": "Ce bouton devrait réinitialiser la pollution sonore.",
+  "Options.GROUP[NoPollution.NoPollution.Mod.PollutionToggles]": "Activer/désactiver les types de pollution",
+  "Options.OPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollution]": "Pollution sonore",
+  "Options.OPTION_DESCRIPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollution]": "Autoriser ou non la pollution sonore (coché signifie que la pollution est active)",
```

Comment:
I'd recommend putting a full stop at the end of `OPTION_DESCRIPTION` keys, as it is common pratice in other mods, renders better, and allows for multiple sentences. This is one of the rare cases where I override the choice of punctuation from the mod author.